docs/manual_configuration.md

```diff
@@ -0,0 +1,201 @@
+## Manual Installation
+Follow these steps if you choose to forgo the generator:
+
```

Comment:
@justin808 This was left over from the original README and I didn't want to delete it, but honestly this file has a huge maintenance cost and is likely to get stale. I vote for picking out anything of value and then getting rid of it.